src/unk_0202055C.c

```diff
@@ -0,0 +1,51 @@
+#include "heap.h"
+#include "unk_0202055C.h"
+
+static Unk0202055C *sUnknown;
+
+void sub_0202055C(u32 a0, HeapID a1) {
+    void *ptr;
+
+    GF_ASSERT(sUnknown == NULL);
+    ptr = AllocFromHeap(a1, sizeof(Unk0202055C));
```

Comment:
Is this intermediate var even necessary to match?

Reply:
Nope, removed